extensions/wikia/FlowTracking/scripts/initCreatePageTracking.js

```diff
@@ -1,18 +1,34 @@
 require([
-	'ext.wikia.flowTracking.createPageTracking', 'wikia.querystring', 'mw', 'jquery', 'wikia.window'
-], function (flowTrackingCreatePage, QueryString, mw, $, window) {
+	'wikia.flowTracking',
+	'ext.wikia.flowTracking.createPageTracking',
+	'wikia.querystring',
+	'mw',
+	'jquery',
+	'wikia.window'
+], function (flowTracking, flowTrackingCreatePage, QueryString, mw, $, window) {
 	function init() {
+		var $wikiaArticle = $('#WikiaArticle'),
+			redLinkFlow = mw.config.get('wgNamespaceNumber') === -1 ?
+			window.wgFlowTrackingFlows.CREATE_PAGE_SPECIAL_REDLINK :
+			window.wgFlowTrackingFlows.CREATE_PAGE_ARTICLE_REDLINK;
```

Comment:
Hmm, this formatting seems strange ( assignment and then parts of ternary condition ).